website/docs/cloud-docs/api-docs/team-tokens.mdx

```diff
@@ -229,3 +229,71 @@ This endpoint supports pagination [with standard URL query parameters](/terrafor
   ]
 }
 ```
+
+## Show a Team Token
+
+A [Team Token](/terraform/cloud-docs/users-teams-organizations/teams#api-tokens) can be displayed for a particular team:
+
+`GET /teams/:team-id/authentication-token`
+
+| Parameter  | Description               |
+| ---------- | ------------------------- |
+| `:team-id` | The ID of the Team.       |
+
+or fetched by its ID:
```

Comment:
Adding a bit more context around this table for clarity. Please feel free to pushback if this doesn't feel good though!
```suggestion
You can also fetch a team's tokens by using the team's ID with the `authentication-tokens/` endpoint:
```

Reply:
this is actually not quire correct, it is the token's ID, not the team's ID which is being used.

Would this work?
```suggestion
You can also fetch a team tokens directly by using the token's ID with the `authentication-tokens/` endpoint:
```

Reply:
That works great, thank you for the context and correction!

Reply:
Once you add your suggestion please feel free to merge this too! 🔥